src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -295,6 +298,7 @@ const BaseComment = (props: Props) => {
             </Media>
             {/* $FlowFixMe */}
             {error ? <ActivityError {...error} /> : null}
+            {!!children && <div className="bcs-Comment-replies">{children}</div>}
```

Comment:
Are we not including the reply button in this PR? Is that functionality we are looking to implement later?

Reply:
Yes we are implementing in a separate PR

Reply:
cool beans